README.md

```diff
@@ -28,7 +28,10 @@ func main() {
         Port:    "8000",
     }
 
-    logger := zerolog.New(os.Stdout).With().Timestamp().Logger()
+	logger, err := baseapp.ConfigureDefaultLogger(config.Logging)
```

Comment:
`config` doesn't contain the field `Logging` in this context. It also looks like the indentation of the new code is off from the existing code.

Reply:
interestingly, `./godelw format` doesn't show any diffs..